definitions/ext-ups/golden_metrics.yml

```diff
@@ -1,23 +1,59 @@
 outputLoadCapacity:
   title: Output load capacity %
   unit: PERCENTAGE
-  query:
-    select: average(kentik.snmp.upsAdvOutputLoad)
-    from: Metric
-    where: "provider = 'kentik-ups'"
+  queries:
+    # APC UPS profiles (default)
+    kentik/apc_ups:
+      select: max(kentik.snmp.upsAdvOutputLoad)
+      from: Metric
+      where: "provider = 'kentik-ups'"
+    # Tripplite UPS profiles
+    kentik/tripplite-ups:
+      select: max(kentik.snmp.upsOutputPercentLoad)
+      from: Metric
+      where: "provider = 'kentik-ups'"
```

Comment:
is this correct? shouldn't be tripplite-ups? same for the previous one

Reply:
It's accurate. Tripplite and the generic UPS have the same metrics as of right now